_extensions/godot_descriptions.py

```diff
@@ -103,10 +103,11 @@ def generate_description(app, pagename, templatename, context, doctree):
     doctree.walkabout(generator)
 
     description = (
-        '<meta name="description" content="' + generator.create_description() + '">\n'
+        '<meta name="description" content="' + generator.create_description() + '" />\n'
     )
 
-    context["metatags"] += description
+    if not re.search('<meta name="description"', context["metatags"]):
```

Comment:
Why not use `'<meta name="description"' in context["metatags"]` to check for presence within the string?

Reply:
Mostly because I'm more familiar with regex (from using `perl`, `ruby` and other languages) and don't know the various string comparison functions in `python`.  If this is a blocker, I'm happy to update the code to whatever is suggested.

Reply:
@dylannorthrup If you are around to make that change that'd be welcome, as thats a bit more pythonic. Besides that, I'll do a final test later today and then merge if nothing else comes up. :) Sorry this has been sitting a bit.

Reply:
I've gone ahead and made the change.

Reply:
My apologies for the delayed response. I had some personal issues that intervened. Thank you for making the change!